feature-libs/product-configurator/rulebased/root/default-rulebased-routing-config.ts

```diff
@@ -12,6 +12,16 @@ export const defaultRulebasedRoutingConfig: RoutingConfig = {
       configureCPQCONFIGURATOR: {
         paths: ['configure/vc/:ownerType/entityKey/:entityKey'],
       },
+      configureCPQCONFIGURATOR_READ_ONLY: {
+        paths: [
+          'configure-overview/vc/:ownerType/entityKey/:entityKey/displayOnly/true',
+        ],
+      },
+      configureOverviewCPQCONFIGURATOR_READ_ONLY: {
+        paths: [
+          'configure-overview/vc/:ownerType/entityKey/:entityKey/displayOnly/TRUE',
```

Comment:
Hi @Platonn,
there are 3 routes:
1. **configureCPQCONFIGURATOR_READ_ONLY**,
2. **configureOverviewCPQCONFIGURATOR_READ_ONLY** and 
3. **configureOverviewCPQCONFIGURATOR** that point to the some path
_configure-overview/vc/:ownerType/entityKey/:entityKey/displayOnly/:displayOnly_

We wonder whether there is a more elegant way to point to the same path instead doing something like:
1. configure-overview/vc/:ownerType/entityKey/:entityKey/displayOnly/:displayOnly
2. configure-overview/vc/:ownerType/entityKey/:entityKey/displayOnly/true or
3. configure-overview/vc/:ownerType/entityKey/:entityKey/displayOnly/TRUE

Could you give us a hint to declare several routes that point to the same path?
Thanks a lot in advance.

Reply:
Hi @Larisa-Staroverova when checking `routing-config.service` it seems that there is not easy way around that. Telling the routes from the path is still used in router.reducer.
So unless Kris has a good idea we could also handle this in configureCartEntry and configureProduct in a way that we generically switch to configureOverview<configuratorType> in case configuratorType has the READ_ONLY postfix.